app/controllers/api/v1/restaurants_controller.rb

```diff
@@ -1,7 +1,12 @@
 require 'overpass_api_ruby'
 require "geocoder"
+#require 'reviews_controller'
+
+
 
```

Comment:
Only need one line of white space, and we should also remove the commented require statement if it's not being used